apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -1,178 +1,179 @@
+          <%= label(f, :publishFrequency, "Update Frequency", class: "label label--required") %>
+          <%= text_input(f, :publishFrequency, class: "input") %>
+          <%= error_tag(f, :publishFrequency) %>
+        </div>
+        <div class="metadata-form__keywords">
+          <%= label(f, :keywords, "Keywords", class: "label") %>
+          <%= text_input(f, :keywords, value: keywords_to_string(input_value(f, :keywords)), class: "input") %>
+          <div class="label label--inline">Separated by comma</div>
+        </div>
+        <div class="metadata-form__last-updated">
+          <%= label(f, :modifiedDate, "Last Updated", class: "label") %>
+          <%= date_input(f, :modifiedDate, class: "input") %>
+        </div>
+        <div class="metadata-form__spatial">
+          <%= label(f, :spatial, "Spatial Boundaries", class: "label") %>
+          <%= text_input(f, :spatial, class: "input") %>
+        </div>
+        <div class="metadata-form__temporal">
+          <%= label(f, :temporal, "Temporal Boundaries", class: "label") %>
+          <%= text_input(f, :temporal, class: "input") %>
+          <%= error_tag(f, :temporal) %>
+        </div>
+        <div class="metadata-form__organization">
+          <%= label(f, :orgTitle, "Organization", class: "label label--required") %>
+          <%= text_input(f, :orgTitle, [class: "input input--text", readonly: true]) %>
+          <%= error_tag(f, :orgTitle) %>
+        </div>
+        <div class="metadata-form__language">
+          <%= label(f, :language, "Language", class: "label") %>
+          <%= select(f, :language, get_language_options(), value: get_language(input_value(f, :language)), class: "select") %>
+        </div>
+        <div class="metadata-form__homepage">
+          <%= label(f, :homepage, "Data Homepage URL", class: "label") %>
+          <%= text_input(f, :homepage, class: "input") %>
+        </div>
+        <div class="metadata-form__format">
+          <%= label(f, :sourceFormat, "Format", class: "label label--required") %>
+          <%= text_input(f, :sourceFormat, [class: "input--text input", readonly: true]) %>
+          <%= error_tag(f, :sourceFormat) %>
+        </div>
+        <div class="metadata-form__level-of-access">
+          <%= label(f, :private, "Level of Access", class: "label label--required") %>
+          <%= select(f, :private, get_level_of_access_options(), class: "select") %>
+          <%= error_tag(f, :private) %>
+        </div>
         <div class="metadata-form__cancel-btn">
           <%= Link.button("Cancel", to: "/", method: "get", class: "btn btn--cancel") %>
         </div>
         <div class="metadata-form__save-btn">
-          <%= Link.button("Next", to: "/", method: "get", id: "next-button", class: "btn btn--next") %>
-          <%= Form.submit("Save", id: "save-button", class: "btn btn--save") %>
+          <%= unless is_nil(@validation_errors) do %>
+            <div class="metadata__error-message">
```

Comment:
Duplicating the error messages at the bottom of the page feels wonky to me. I'd take it out unless Jake asked for it.

Reply:
These are the error messages that come from the DatasetValidator, not the form errors. We'll double check the UX with Jake in the exit, but expect that more work will be done on this with the validation card (#328)